.github/workflows/_nodejs.condo.core.tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         env:
           REGISTRY: ${{ secrets.DOCKER_REGISTRY }}/doma/utils/
         run: |
-          docker-compose up -d redis postgresdb
+          docker-compose up -d valkey-cluster postgresdb
```

Comment:
How about adding some aliases? Like `docker compose up -d dbs`?

Reply:
Btw, dash syntax is deprecated....Use `docker compose` instead of `docker-compose`